lib/destination-tile/index.js

```diff
@@ -12,9 +12,12 @@ class DestinationTile extends Component {
     const { image, title, text } = sections[0];
     let previewTextArray = text.split('').slice(0, 292);
     if (previewTextArray[previewTextArray.length - 1] === ' ') {
-      previewTextArray = previewTextArray.slice(0, previewTextArray.length - 1);
+      previewTextArray = previewTextArray.slice(0, previewTextArray.length - 1); // remove unecessary whitespace at the end of the string
     }
-    const previewText = previewTextArray.slice(0, previewTextArray.length - 1).join('') + '...</div>';
+    if (previewTextArray.slice(0, 5).join('').indexOf('<div>') === -1) {
+      previewTextArray = ['<div>', ...previewTextArray]; // if the paragraph doesn't start with a <div> then add one so the html is valid
```

Comment:
The code and the comment don't quite align here. The comment talks about *starting* with a `<div>`, but the code is checking whether it *contains* a `<div>`.